### PR TITLE
fixed some typos and technical terms

### PR DESCRIPTION
**Pull request by Achele**
issue number #41 Fixed typo and ammended some technical terms

### CODE REVIEW

**Comment by Achele** [comment]
issue number https://github.com/jupyterhub/jupyterhub/pull/41 Fixed typo and ammended some technical term

**Review by GeorgianaElena — COMMENTED** [review]
Thank you @Achele 🌻 ! I left an inline comment about the `passed` -> `parsed` change.

File: docs/source/rbac/tech-implementation.md [hunk 1]
```diff
@@ -78,15 +78,15 @@ Figure 1. Resolving roles and scopes during API token request
...
 
 When an API request is performed, the requesting API token's scopes are again intersected with its owner's (yellow box in {ref}`Figure 2 <api-request-chart>`) to ensure the token does not grant more permissions than its owner has at the request time (e.g., due to changing/losing roles).
-If the owner's roles do not include some scopes of the token's scopes, only the _intersection_ of the token's and owner's scopes will be used. For example, using a token with scope `users` whose owner's role scope is `read:users:name` will result in only the `read:users:name` scope being passed on. In the case of no _intersection_, an empty set of scopes will be used.
+If the owner's roles do not include some scopes of the token's scopes, only the _intersection_ of the token's and owner's scopes will be used. For example, using a token with scope `users` whose owner's role scope is `read:users:name` will result in only the `read:users:name` scope being parsed on. In the case of no _intersection_, an empty set of scopes will be used.
```
[the remark this answers is not in the document]
Comment (Achele) [reply docs/source/rbac/tech-implementation.md:81]: Thank you @GeorgianaElena for the correction. I have noted and updated the changes

**Comment by GeorgianaElena** [comment]
Thank you @Achele! 🌻

**Pull request merged**